### ISSUE TITLE
[Usable – Make Code – Publish Project] Narrator announces 'Project URL' information twice while navigating via tab key under 'Share project' pop-up.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3601]
**User Experience:**
Screen reader user will be confused, if narrator is reading same information two times and also it will take time to navigate another control over the screen.

**Note**: User credentials should NOT be included in the bug.

**Repro Steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Share" option appearing in header section and select it.
6. Navigate to "Publish project" under  "Share Project" dialog and select it.
7. Navigate to all the control appearing like QR Code, Embed(Code, Editor and Simulator).
8. Verify that Narrator announces 'Project URL' information twice while navigating via tab key or not.

**Actual Result:**
Narrator announces 'Project URL' information twice while navigating via tab key under 'Share project' pop-up.

**Expected Result:**
Narrator should announce 'Project URL' information only once while navigating via tab key under 'Share project' pop-up.

**Note:**
1. This issue does not exists on Firefox & Chrome.
2. Same issue exists under 'Report Abuse' pop-up window.

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0)
Screen Reader: Narrator

**Comment by Shalini3457** [comment]
Labels:
#A11yUsable, #Accessibility, #A11ySev4, #HCL, #Benchmark-HCL-MakeCode-OCT2020, #Win10-Edge(Chromium)

[Usable_Publish projects_Twice URL information.zip](https://github.com/microsoft/pxt-microbit/files/5499242/Usable_Publish.projects_Twice.URL.information.zip)

**Comment by jwunderl** [comment]
no repro, appears fixed in edge update
